applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf

```diff
@@ -15,3 +15,6 @@ CONFIG_BT_MAX_CONN=2
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
 CONFIG_BT_BUF_ACL_TX_SIZE=251
 CONFIG_BT_BUF_ACL_RX_SIZE=502
+
+# Workaround FEM issue seen on older Thingy:53 revisions
+CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA=10
```

Comment:
Does it affect only our application or other Thingy:53 applications should be aligned too?

Reply:
The thing is the default setting is supposedly correct and the problem is that we use too old Thingy:53 revisions, so I don't want to take the responsibility of changing it for all teams.

Reply:
Ok and what if we will use the other applications with our old Thingys in CI to verify DFU? Does it mean they may fail to connect too?

Reply:
I think then we can change this flag in our test. I think other applications may also struggle from that depending on which Thingy revision they are run on.